.github/workflows/test-application.yaml

```diff
@@ -193,3 +193,104 @@ jobs:
+            matrix:
+                include:
+                    - php-version: '7.4'
+                      mysql-version: '8.0'
+                      phpcr-transport: 'jackrabbit'
+                      dependency-versions: 'highest'
+                      php-extensions: 'ctype, iconv, intl, mysql, pdo_mysql, php_fileinfo, imagick'
+                      tools: 'composer'
+
+        steps:
+            - name: Checkout project
+              uses: actions/checkout@v2
+
+            - name: Install and configure Node.js
+              uses: actions/setup-node@v2
+              with:
+                  node-version: '14'
+
+            - name: Install and configure PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php-version }}
+                  extensions: ${{ matrix.php-extensions }}
+                  tools: ${{ matrix.tools }}
+                  ini-values: 'memory_limit=-1'
+
+            - name: Install and configure MySQL
+              uses: shogo82148/actions-setup-mysql@v1.0.1
+              with:
+                  mysql-version: ${{ matrix.mysql-version }}
+                  root-password: 'root'
+                  my-cnf: |
+                      [mysqld]
+                      default-authentication-plugin=mysql_native_password
+
+            - name: Cache jackrabbit runnable
+              uses: actions/cache@v2
+              id: jackrabbit-cache
+              if: ${{ matrix.phpcr-transport == 'jackrabbit' }}
+              with:
+                  path: downloads
+                  key: ${{ runner.os }}-jackrabbit-${{ env.JACKRABBIT_VERSION }}
+
+            - name: Download jackrabbit
+              if: ${{ matrix.phpcr-transport == 'jackrabbit' && steps.jackrabbit-cache.outputs.cache-hit != 'true' }}
+              run: |
+                  mkdir downloads
+                  cd downloads
+                  curl -O http://archive.apache.org/dist/jackrabbit/${{ env.JACKRABBIT_VERSION }}/jackrabbit-standalone-${{ env.JACKRABBIT_VERSION }}.jar
+                  cd -
+
+            - name: Run jackrabbit
+              if: ${{ matrix.phpcr-transport == 'jackrabbit' }}
+              run: powershell -command "Start-Process -FilePath java -ArgumentList '-jar downloads/jackrabbit-standalone-${{ env.JACKRABBIT_VERSION }}.jar'"
+
+            - name: Require jackrabbit dependencies
+              if: ${{ matrix.phpcr-transport == 'jackrabbit' }}
+              run: |
+                  composer remove jackalope/jackalope-doctrine-dbal --dev --no-interaction --no-update
+                  composer require jackalope/jackalope-jackrabbit:^1.4  --no-interaction --no-update
+
+            - name: Install composer dependencies
+              uses: ramsey/composer-install@v1
+              with:
+                  dependency-versions: ${{ matrix.dependency-versions }}
+
+            - name: Build sulu
+              run: bin/adminconsole sulu:build dev --no-interaction
+
```

Comment:
should we include `Build container`, `Lint container` and `Lint code` here too?

```suggestion

            - name: Build container
              run: |
                  bin/adminconsole cache:clear --env dev
                  bin/websiteconsole cache:clear --env dev
                  bin/adminconsole cache:clear --env test
                  bin/websiteconsole cache:clear --env test
                  bin/adminconsole cache:clear --env stage
                  bin/websiteconsole cache:clear --env stage
                  bin/adminconsole cache:clear --env prod
                  bin/websiteconsole cache:clear --env prod

            - name: Lint container
              run: |
                  bin/adminconsole lint:container --env dev
                  bin/websiteconsole lint:container --env dev
                  bin/adminconsole lint:container --env test
                  bin/websiteconsole lint:container --env test
                  bin/adminconsole lint:container --env stage
                  bin/websiteconsole lint:container --env stage
                  bin/adminconsole lint:container --env prod
                  bin/websiteconsole lint:container --env prod

            - name: Lint code
              run: |
                  bin/adminconsole doctrine:ensure-production-settings --env prod
                  bin/adminconsole doctrine:schema:validate
                  bin/adminconsole lint:twig templates
                  bin/adminconsole lint:yaml config

```

Reply:
okay for me but not really required as this linting are not platform specific